tests/benchmarks/micro_benchmarks/test_disk_performance.py

```diff
@@ -0,0 +1,493 @@
+                curr_test_magic += 1
+        benchmark = benchmark_class(benchmark_name, parameters=param_str)
+
+        # Check basic information
+        assert (benchmark)
+        ret = benchmark._preprocess()
+        assert (ret is True)
+        assert (benchmark.return_code == ReturnCode.SUCCESS)
+        assert (benchmark.name == 'disk-performance')
+        assert (benchmark.type == BenchmarkType.MICRO)
+
+        # Check command list
+        # 2 files * (2 preconditions + 2 io_patterns * 2 io_types) = 12 commands
+        assert (12 == len(benchmark._commands))
+
+        # Check parameter assignments
+        command_idx = 0
+
+        for filename in filenames:
+            curr_test_magic = init_test_magic
+
+            # Sequential precondition
+            assert ('--filename=%s' % filename in benchmark._commands[command_idx])
+            command_idx += 1
+            # Random precondition
+            assert ('--filename=%s' % filename in benchmark._commands[command_idx])
+            assert ('--runtime=%d' % curr_test_magic in benchmark._commands[command_idx])
+            curr_test_magic += 1
+            command_idx += 1
+            # Seq/rand read/write
+            for io_pattern in ['seq', 'rand']:
+                for io_type in ['read', 'write']:
+                    assert ('--filename=%s' % filename in benchmark._commands[command_idx])
+                    fio_rw = '%s%s' % (io_pattern if io_pattern == 'rand' else '', io_type)
+                    assert ('--rw=%s' % fio_rw in benchmark._commands[command_idx])
+                    assert ('--ramp_time=%d' % curr_test_magic in benchmark._commands[command_idx])
+                    curr_test_magic += 1
+                    assert ('--runtime=%d' % curr_test_magic in benchmark._commands[command_idx])
+                    curr_test_magic += 1
+                    assert ('--iodepth=%d' % curr_test_magic in benchmark._commands[command_idx])
+                    curr_test_magic += 1
+                    assert ('--numjobs=%d' % curr_test_magic in benchmark._commands[command_idx])
+                    curr_test_magic += 1
+                    command_idx += 1
+
+    def test_disk_performance_result_parsing(self):
+        """Test disk-performance benchmark result parsing."""
+        benchmark_name = 'disk-performance'
+        (benchmark_class,
+         predefine_params) = BenchmarkRegistry._BenchmarkRegistry__select_benchmark(benchmark_name, Platform.CPU)
+        assert (benchmark_class)
+        benchmark = benchmark_class(benchmark_name, parameters='')
+        assert (benchmark)
+        ret = benchmark._preprocess()
+        assert (ret is True)
+        assert (benchmark.return_code == ReturnCode.SUCCESS)
+        assert (benchmark.name == 'disk-performance')
+        assert (benchmark.type == BenchmarkType.MICRO)
+
+        # Positive case - valid raw output.
+        test_raw_output = """
+{
```

Comment:
read it from a test data file to separate from code?

Reply:
I just followed the style in `test_gemm_flops_performance` and `test_cuda_memory_bw_performance`.